hubspot.com.hubspot-txt.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "hubspot.com",
+  "providerName": "HubSpot",
+  "serviceId": "hubspot-txt",
+  "serviceName": "HubSpot Site",
+  "version": 1,
+  "logoUrl": "https://domainconnect.org/wp-content/uploads/2017/02/HubSpot-768x223.png",
+  "description": "Enables a domain to work with HubSpot",
+  "variableDescription": "txtValue - txt record data value",
+  "hostRequired": true,
```

Comment:
Is this template only for subdomains (hostRequired) or also for APEX?

Reply:
Good catch. we should set hostRequired to false. this template is used for both subdomain and domain. we ask our customer to add a TXT record to verify the ownership. for example
text123.hubspot.com
text456.sub.hubspot.com